apps/src/templates/HideToolbarHelper.jsx

```diff
@@ -43,8 +43,11 @@ export default class HideToolbarHelper extends React.Component {
     }
   }
 
-  isiOS13() {
-    return navigator.userAgent.indexOf('iPhone OS 13') !== -1;
+  isCompatibleiOS() {
+    return (
+      navigator.userAgent.indexOf('iPhone OS 13') !== -1 ||
+      navigator.userAgent.indexOf('iPhone OS 14') !== -1
```

Comment:
Would it make sense to split the version number out and check for versions `>=13` ?

Reply:
I think refactoring makes sense if this changes again.  I don't want to presume behaviour in future versions, though, since this UI could change again...